File: C/URI_judge_1010.c
#include <stdio.h>

int main(){
    int x1,x2,y1,y2;
    double z1,z2,t;
    scanf("%d %d %lf",&x1,&y1,&z1);
    scanf("%d %d %lf",&x2,&y2,&z2);
    t = (y1 * z1) + (y2 * z2);
    printf("VALOR A PAGAR: R$ %.2lf\n",t);
}

File: Python/URI_judge_1010.py
n1 = input().split()
n2 = input().split()

c1, qtd1, v1 = n1
c2, qtd2, v2 = n2

tt = (int(qtd1)) * (float(v1)) + (int(qtd2)) * (float(v2))

print(f"VALOR A PAGAR: R$ {tt:.2f}")